File: 1 - BEGINNER/1036 - Fórmula de Bhaskara.c
#include <stdio.h>
#include <math.h>
 
int main() {
 
    double a, b, c, raiz_um, raiz_dois, delta;
    
    scanf("%lf %lf %lf", &a, &b, &c);
    
    delta = b * b - 4 * a * c;
    
    if(delta >= 0 && a > 0){
        
        raiz_um = (-b + sqrt(delta)) / (2 * a);
        raiz_dois = (-b - sqrt(delta)) / (2 * a);
        printf("R1 = %.5lf\n", raiz_um);
        printf("R2 = %.5lf\n", raiz_dois);
        
    } else{
        printf("Impossivel calcular\n");
    }

    return 0;
}

File: 1 - BEGINNER/Java/1036 - Fórmula de Bhaskara.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        double A = input.nextDouble();
        double B = input.nextDouble();
        double C = input.nextDouble();
        double delta = (B * B) - (4 * A * C);
        double R1 = (-B + Math.sqrt(delta)) / (2 * A);
        double R2 = (-B - Math.sqrt(delta)) / (2 * A);
        
        if(A == 0 || delta < 0){
            System.out.println("Impossivel calcular");
        } else{
            System.out.printf("R1 = %.5f\n", R1);
            System.out.printf("R2 = %.5f\n", R2);
        }
        
        input.close();
 
    }
 
}